postgres-appliance/scripts/configure_spilo.py

```diff
@@ -580,6 +590,7 @@ def get_placeholders(provider):
     placeholders.setdefault('CLONE_TARGET_INCLUSIVE', True)
 
     placeholders.setdefault('LOG_GROUP_BY_DATE', False)
+    placeholders.setdefault('LOG_SHIP_HOURLY', False)
```

Comment:
setting default as false. I would not mention the variable in the ENVIRONMENT.rst file because it has only an internal purpose (could someone still override it?)

Reply:
Changed my mind. It's a lot easier to just mention the flag in a Postgres Operator manifest than coming up with a schedule.